featuretools/tests/entityset_tests/test_dask_es.py

```diff
@@ -98,38 +104,45 @@ def test_add_last_time_indexes():
                                           pd.to_datetime('2019-01-01 12:49'),
                                           pd.to_datetime('2017-08-25 04:53')]})
     transactions_dask = dd.from_pandas(transactions, npartitions=2)
-    transactions_vtypes = {
-        "id": ft.variable_types.Id,
-        "session_id": ft.variable_types.Id,
-        "amount": ft.variable_types.Numeric,
-        "time": ft.variable_types.DatetimeTimeIndex,
+
+    transactions_logical_types = {
+        "id": ltypes.Integer,
+        "session_id": ltypes.Integer,
+        "time": ltypes.Datetime,
+        "amount": ltypes.Double
     }
+    transactions_semantic_tags = {'session_id': 'foreign_key'}
```

Comment:
probably a similar thought as I was having with the earlier use of the `foreign_key` semantic tag. Since these tests don't hinge on the `foreign_key` tag, it seems like we shouldn't be adding it (will also have to update the koalas es tests accordingly)

Reply:
however, I do think it might be useful to confirm that the `foreign_key` tag gets added as a part of relationship creation in these tests (even less reason to add the tag directly at add_dataframe), so I'm going to add checks that confirm that the tag gets added

Reply:
Sounds good!